### PR TITLE
add background for theme context

### LINKED ISSUE #2325: DataTable - Background color on DataTable header incorrect color
The Design System site is showing a bug on the DataTable header. Here's an image of what it's showing:
Screen Shot 2022-02-07 at 8 49 57 AM

[Please align to Figma](https://www.figma.com/file/XEoIWIlLfcARRIFhkpFCch/HPE-DataTable?node-id=2390%3A5274). Thanks :)

### PR DESCRIPTION
**Pull request by britt6612**
[Deploy Preview](https://deploy-preview-2348--keen-mayer-a86c8b.netlify.app/components/datatable?q=data)

#### What does this PR do?
This PR fixes the `datatable` header color by adding `backgroud` to `theme.context`
#### Where should the reviewer start?
Example.js
#### What testing has been done on this PR?

In addition to the feature you are implementing, have you checked the following:

**General UX Checks**
- [x] Small, medium, and large screen sizes
- [x] Cross-browsers (FireFox, Chrome, and Safari)
- [x] Light & dark modes
- [x] All hyperlinks route properly

**Accessibility Checks**
- [x] Keyboard interactions
- [x] Screen reader experience
- [x] Run WAVE accessibility plugin (Chrome)

**Code Quality Checks**
- [x] Console is free of warnings and errors
- [x] Passes E2E commit checks
- [ ] Visual snapshots are reasonable

#### Any background context you want to provide?
The `Header` of the `pinned` DataTable turned into a grey color. 
When the `Theme.Context` was added to the Example.js was when this regression happened. 
With the  `Theme.Context` wrapper it is looking at the theme background color instead of the 
background color of the example wrapper which is the `background-back`

#### What are the relevant issues?
closes #2325 
#### Should this PR be mentioned in Design System updates?
no
#### Is this change backwards compatible or is it a breaking change?
backwards compatible

### CODE REVIEW

**Review by taysea — APPROVED** [review]
Looks good code wise, but can we add a comment explaining why we need to do this?

**Comment by britt6612** [comment]
> Looks good code wise, but can we add a comment explaining why we need to do this?

yes I will add

File: aries-site/src/layouts/content/Example/Example.js [hunk 1]
```diff
@@ -180,7 +180,10 @@ export const Example = ({
         width={width}
         ref={inlineRef}
       >
-        <ThemeContext.Extend value={scaledTheme || theme}>
+        {/* need to overrid theme background so its not resetting */}
```
Comment (halocline) [inline aries-site/src/layouts/content/Example/Example.js:183]: overrid --> override

Does this make the intent any more clear? As currently written, I got the "what is happening" but still had trouble following the "why we want it".

```suggestion
        {/* prevent theme from overriding the desired background color */}
```

File: aries-site/src/layouts/content/Example/Example.js [hunk 2]
```diff
@@ -180,7 +180,10 @@ export const Example = ({
         width={width}
         ref={inlineRef}
       >
-        <ThemeContext.Extend value={scaledTheme || theme}>
+        {/* prevent theme from overriding the desired background color */}
+        <ThemeContext.Extend
+          value={{ ...(scaledTheme || theme), background: 'background' }}
```
Comment (taysea) [inline aries-site/src/layouts/content/Example/Example.js:185]:
```suggestion
          value={{ ...(scaledTheme || theme), background: 'background-front' }}
```
Think we need to do `background-front` instead.

Comment (taysea) [reply aries-site/src/layouts/content/Example/Example.js]: For the way dark vs light mode differ

**Comment by britt6612** [comment]
> I am seeing the data table header as a different color. Is that the intent?
> 
> Screen Shot 2022-02-11 at 11 28 20 AM

good catch it should be `background-front`

**Pull request merged**